### PR TITLE
Update documents

### PR DESCRIPTION
**Pull request by borja-munoz**
I've updated two README.md files and the DEVELOPERS.md.

In my opinion we have too many files with documentation (3 README.md files and the DEVELOPERS.md file). Maybe this is the best option but I think this could be confusing.

I haven't altered too much the original content on purpose. I think the main documentation should be the one within the docs folder. The other README.md files are more for explaining the purpose of the template (root folder) and for explaining how the template has been created and how you can work with it (template folder).

Please let me know if you think it is better to follow a different approach.

### CODE REVIEW

File: template/README.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ You can learn more in the [Create React App documentation](https://facebook.gith
 
 To learn React, check out the [React documentation](https://reactjs.org/).
 
+To learn more about the template, you can access additional documentation [here](docs/README.md).
```
Comment (alasarr) [inline template/README.md:46]:
```suggestion
```

File: template/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [CARTO](https://carto.com/developers/create-react-app) template.
```
Comment (alasarr) [inline template/README.md:2]:
```suggestion
* [What is it](docs/README.md#what-is-it)?
* [Architecture](docs/README.md#architecture).
* [How to guides](docs.README.md#how-to-guides).
* [FAQ](docs.README.md#frequently-asked-questions-faq).
```

File: template/README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
...
```
Comment (alasarr) [inline template/README.md:1]: https://carto.com/developers/create-react-app is not available yet. Let's include when it's available

**Review by alasarr — CHANGES_REQUESTED** [review]
Remove also yarn test as we don't have test right now

**Comment by alasarr** [comment]
>In my opinion we have too many files with documentation (3 README.md files and the DEVELOPERS.md file). Maybe this is the best option but I think this could be confusing.

Agree

Let's merge `DEVELOPERS.md` in `/README.md` inside 

`template/README.md` is the most important because is the one that users see when they install the template.

`template/docs/README.md` is the initial point of the doc and we want a separated folder to grow, so it makes sense to me

File: template/README.md (hunk 3 above)
Comment (borja-munoz) [reply template/README.md]: Link removed

**Review by alasarr — APPROVED** [review]

**Pull request merged**